santosfamilyfoundation/RuinaMain
# [RUIN-236] Add save button to the bottom of each form section

# Description
Previously to go back to the home screen to edit the other forms, users had to click on the back arrow. Back arrows are typically associated with canceling inputs and not saving data. Now the user can choose the back arrow or the done button to navigate back to the home screen. The done button gives the user a safety button because it gives them security that the data will be saved.

# Images

## Before
![image](https://user-images.githubusercontent.com/42981026/141536071-695362c2-b796-41fe-8136-496568a7247f.png)

## After
![image](https://user-images.githubusercontent.com/42981026/141535823-b5e283c0-22ac-4e2b-abd7-8bb032db0837.png)

# Testing

1. Open a report
2. Navigate to any form
3. "Done" button should appear at the bottom
4. Click on the "Done" button
5. Should be navigated back to the Home screen
